### ISSUE TITLE
Status Update?

### ISSUE THREAD

**Issue by dimo414** [protocolbuffers/protobuf-javascript#175]
The [README notes](https://github.com/protocolbuffers/protobuf-javascript#project-status-as-of-september-2022):

> This project is currently in a somewhat broken state that we are working to rectify. We expect to have updated releases by the end of September 2022.

A year later, is there any status update? Is the state less broken? Can more details be shared about what is/was broken and what is/was being planned?

**Comment by ryanberckmans** [comment]
Use this new, modern, excellent library instead https://github.com/bufbuild/protobuf-es

**Comment by dimo414** [comment]
Thanks @ryanberckmans, I'm looking for an authoritative reply from the maintainers, not an alternative library to consider.

**Comment by yurifedoseev** [comment]
Up to the question. Will this project be maintained?

**Comment by dibenede** [comment]
Hi folks. This project is still maintained, but staffing is minimal. I have a status update for the readme in flight.

> The [README notes](https://github.com/protocolbuffers/protobuf-javascript#project-status-as-of-september-2022):
> 
> > This project is currently in a somewhat broken state that we are working to rectify. We expect to have updated releases by the end of September 2022.
> 
> A year later, is there any status update? Is the state less broken? Can more details be shared about what is/was broken and what is/was being planned?

The project is in a working state. In September 2022, the project was broken in the sense it did not build due to changes from being refactored out of the main protobuf repo. We resolved that and, with help from contributors, got cross platform going.

I think the single biggest issue users have with protobuf-javascript is code size. We accepted a contribution to try to turn on the Closure Compiler's advanced optimization mode. However, we were alerted shortly thereafter that it broke our Node.js users. This was triaged to be from some missing export annotations, if I recall and did a release to disable the optimizations.

While we added a bunch more annotations, the fundamental issue is we aren't comfortable with the existing test coverage for Node.js to try to release with optimizations enabled again. They can, however, still be turned on by users.
